python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -488,4 +489,9 @@ def process_graph(args):
     argparser.add_argument("--ext_mem_feat_size", type=int, default=64,
                            help="The minimal number of feature dimensions that features " + \
                                    "can be stored in external memory.")
+    argparser.add_argument("--save_id_mapping",
```

Comment:
do we need to expose this option? ID mapping should always be saved.
if the argument with bool value, we should make it consistent. `--skip_nonexist_edges` and `--add_reverse_edges` use `action='store_true'`. i think all boolean arguments should work in the same way.

Reply:
I agree we should always save the mapping. Let me remove this argument. We do not need it.